src/sentry_plugins/bitbucket/endpoints/webhook.py

```diff
@@ -23,6 +23,11 @@
 # Bitbucket Cloud IP range:
 # https://confluence.atlassian.com/bitbucket/manage-webhooks-735643732.html#Managewebhooks-trigger_webhookTriggeringwebhooks
 BITBUCKET_IP_RANGE = ipaddress.ip_network(u'104.192.136.0/21')
+BITBUCKET_IPS = [
+    ipaddress.ip_address(u'34.198.203.127'),
```

Comment:
We don't even need to make these `ipaddress` objects. Can just compare with: `request.META['REMOTE_ADDR'] in BITBUCKET_IPS` if they're all strings.